src/View/_navbar.html.twig

```diff
@@ -5,6 +5,9 @@
         <div class="line3"></div>
     </div>
     <ul class="nav-links">
+    <div>
+        <img src="/assets/images/logo.png" class="logo" alt ="Logo LPEV">
```

Comment:
avoid "logo", "image"... in alt text: it should contain significant description. you can check: https://www.w3.org/WAI/tutorials/images/decision-tree/